txpool/pool.go

```diff
@@ -601,16 +602,16 @@ func (p *TxPool) IsLocal(idHash []byte) bool {
 func (p *TxPool) AddNewGoodPeer(peerID types.PeerID) { p.recentlyConnectedPeers.AddPeer(peerID) }
 func (p *TxPool) Started() bool                      { return p.started.Load() }
 
-func (p *TxPool) best(n uint16, txs *types.TxsRlp, tx kv.Tx, onTopOf, availableGas uint64, toSkip [][32]byte) (bool, [][32]byte, error) {
+func (p *TxPool) best(n uint16, txs *types.TxsRlp, tx kv.Tx, onTopOf, availableGas uint64, toSkip mapset.Set[[32]byte]) (bool, int, error) {
 	// First wait for the corresponding block to arrive
 	if p.lastSeenBlock.Load() < onTopOf {
-		return false, nil, nil // Too early
+		return false, 0, nil // Too early
 	}
 
 	txs.Resize(uint(cmp.Min(int(n), len(p.pending.best.ms))))
 	j := 0
 	var toRemove []*metaTx
-	yielded := make([][32]byte, 0)
+	count := 0
```

Comment:
`j` & `count` are the same, aren't they? Probably better to just rename `j` to `count`.

Reply:
Good shout, sorted